src/main/scala-2.12/org/apache/james/gatling/control/RecipientFeeder.scala

```diff
@@ -0,0 +1,24 @@
+package org.apache.james.gatling.control
+
+import org.apache.james.gatling.jmap.RecipientAddress
+
+object RecipientFeeder {
+
+  type RecipientFeeder = Array[Map[String, String]]
+
+  val RecipientSessionParam = "recipient"
```

Comment:
RECIPIENT_SESSION_PARAM
i'm ok with recipientSessionParam too.
But we mistake it with a type name